contracts/UniswapV3Staker.sol

```diff
@@ -296,6 +341,19 @@ contract UniswapV3Staker is IUniswapV3Staker, IERC721Receiver, Multicall {
                 tickLower,
                 tickUpper
             );
+
+        incentives[incentiveId].totalSecondsClaimedX128 += secondsInPeriodX128;
+        incentives[incentiveId].totalRewardUnclaimed = uint128(
+            SafeMath.sub(incentive.totalRewardUnclaimed, reward)
+        );
+        stakes[params.tokenId][incentiveId].secondsPerLiquidityInitialX128 =
+            secondsPerLiquidityInsideX128 +
+            1;
```

Comment:
why +1?

Reply:
I didn't want to double count the current second at the time of this claim. But actually this isn't "time seconds" but "liquidity seconds", so I was confused...